tests/integration/test_common_metadata_pyarrow.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
```

Comment:
not sure where these files come from. Are you sure they should be here?

Reply:
@crepererum, These are the tests we have pulled from units tests of kartothek which was failed with the errors for pyarrow latest builds and we are adding here for the integration tests, we need to add/remove anything here?